awatif-ui/src/Viewer.ts

```diff
@@ -16,13 +16,17 @@ import { ElementResults } from "./objects/ElementResults";
 import { NodeResults } from "./objects/NodeResults";
 
 export function Viewer(model: ModelState, settings: SettingsState) {
+  // config
+  let playerHeight = 0;
```

Comment:
There is no need to change the height of the viewer if you used the position attribute in CSS like so:
```
.player {
  position: absolute
  bottom: 0px
}
```
It is cleaner to handle the positioning in the CSS file where it belong

Reply:
Yeah this is much cleaner. I've pushed those changes.